### PR TITLE
Added new recipe - Parro

### PR DESCRIPTION
**Pull request by richardbeumer**
#### Pre-flight Checklist

Please ensure you've completed all of the following.

- [X] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I updated the version package [Updating](https://github.com/ferdium/ferdium-recipes/blob/main/docs/updating.md#4-updating-the-version-number) 

#### Description of Change

Added recipe for Parro, a ductch communication platform for parents to comminicate with their kids teachers

### CODE REVIEW

File: recipes/parro/webview.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    const FerdiumData = document.querySelector('#FerdiumMessages').dataset;
+    if (FerdiumData) {
+      direct = FerdiumData.direct;
+      indirect = FerdiumData.indirect;
+    }
+
+    Ferdium.setBadge(direct, indirect);
+  }
+
+  Ferdium.injectCSS(_path.join(__dirname, 'darkmode.css'));
```
Comment (vraravam) [inline recipes/parro/webview.js:20]: you should not remove the injection of the `service.css` - please resurrect it

**Review by vraravam — CHANGES_REQUESTED** [review]

File: recipes/parro/webview.js (hunk 1 above)
Comment (richardbeumer) [reply recipes/parro/webview.js]: removed darkmode.css completly since there is none anymore.

File: recipes/parro/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Parro for Ferdium
```
Comment (vraravam) [inline recipes/parro/README.md:1]: We don't really need this file. Please remove it.

File: recipes/parro/package.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "id": "parro",
+  "name": "Parro",
+  "version": "1.0.1",
```
Comment (vraravam) [inline recipes/parro/package.json:4]: pls force this to 1.0.0 (any new recipe that we induct, we would like to start off with 1.0.0 version

File: recipes/parro/webview.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+module.exports = Ferdium => {
```
Comment (vraravam) [inline recipes/parro/webview.js:1]: the lines that were generated for loading/importing the `service.css` are missing. please resurrect

**Review by vraravam — CHANGES_REQUESTED** [review]

File: recipes/parro/package.json (hunk 3 above)
Comment (richardbeumer) [reply recipes/parro/package.json]: done

File: recipes/parro/README.md (hunk 2 above)
Comment (richardbeumer) [reply recipes/parro/README.md]: done

File: recipes/parro/webview.js (hunk 4 above)
Comment (richardbeumer) [reply recipes/parro/webview.js:7]: fixed

**Review by vraravam — APPROVED** [review]

**Comment by vraravam** [comment]
@all-contributors please add @richardbeumer for code

**Pull request merged**